docs/computing/interconnect.md

```diff
@@ -1,4 +1,6 @@
- 
+[slingshot11-technical-white-paper]: https://www.hpe.com/us/en/collaterals/collateral.a50002368.HPE-Slingshot-E2-80-93-The-Interconnect-for-the-Exascale-Era-technical-white-paper.html?rpv=cpf&parentPage=/us/en/products/compute/hpc/slingshot-interconnect.html
+[slingshot11-spec-sheet]: https://www.hpe.com/psnow/doc/a50002546enw.pdf?jumpid=in_pdp-psnow-qs
 
- A technical description of the Slingshot 11 interconnect
+All nodes in LUMI are connected using the HPE Slingshot 11 interconnect.
 
```

Comment:
I agree that it's good to have something about this, at least links to where one finds information. (Maybe at some point it would be nice to add a short text description on this page, but for starters I think this is nice.)

Reply:
Excellent. I will keep this page as is for now. We have a future work task on adding more information to this page.